features/plugin-activate.feature

```diff
@@ -62,6 +62,27 @@ Feature: Activate WordPress plugins
       Plugin 'user-switching' activated.
       """
 
+  Scenario: Activating a plugin with no network wide option passes down correct types
+    Given a wp-content/plugins/example-plugin.php file:
+      """
+      <?php
+      // Plugin Name: Example Plugin
+
+      function example_plugin_activate( bool $network_wide = false ) {
+        // Doesn't matter what we do here, we just need a function definition to check the type
+        return;
+      }
+
+      register_activation_hook( __FILE__, 'example_plugin_activate' );
+      """
+
+    When I run `wp plugin activate example-plugin`
+    Then STDOUT should be:
+      """
+      Plugin 'example-plugin' activated.
+      Success: Activated 1 of 1 plugins.
+      """
+
```

Comment:
```suggestion
      And STDERR should be empty

```

Just for completeness, to verify there are no errors

Reply:
Of course. Thanks again! I have applied this change now.